source/includes/_invoice.md

```diff
@@ -2658,6 +2658,357 @@ Note that for `SUBSCRIPTION_ACTION`, there are 2 dates to take into account:
 
 If successful, returns a status code of 200 and an invoice resource object.
 
+## Invoice Groups
+
+An invoice group is a group of invoices. An invoice group plugin can be used to split an invoice into multiple invoices. Such split invoices belong to a single invoice group. 
+
+Note that Kill Bill does not provide any reference implementation for an invoice group plugin. So anyone wishing to use this feature would need to create an invoice plugin and implement the [InvoicePluginApi#getInvoiceGrouping](https://github.com/killbill/killbill-plugin-api/blob/20e10d463b67b0434e1b9f4dd62ed34ebe84bd9b/invoice/src/main/java/org/killbill/billing/invoice/plugin/api/InvoicePluginApi.java#L41) method. We provide a [demo plugin](https://github.com/killbill/killbill-invgrp-demo) that demonstrates how an invoice group plugin can be implemented. When such a plugin is configured, an [invoice group run](https://killbill.github.io/slate/#invoice-trigger-an-invoice-group-run) would result in the generation of multiple invoices.
+
+
+### Trigger an invoice group run
+
+Triggers an invoice group run for the associated account. This would result in the creation of split invoices only if an invoice group plugin is configured. Otherwise, this endpoint behaves the same way as the [Trigger an invoice run](https://killbill.github.io/slate/#invoice-trigger-an-invoice-run) endpoint.
+
+**HTTP Request** 
+
+`POST http://127.0.0.1:8080/1.0/kb/invoices/group`
+
+> Example Request:
+
+```shell
+curl -v \
+    -X POST \
+    -u admin:password \
+    -H "X-Killbill-ApiKey: bob" \
+    -H "X-Killbill-ApiSecret: lazar" \
+    -H "Content-Type: application/json" \
+    -H "Accept: application/json" \
+    -H "X-Killbill-CreatedBy: demo" \
+    -H "X-Killbill-Reason: demo" \
+    -H "X-Killbill-Comment: demo" \
+    "http://127.0.0.1:8080/1.0/kb/invoices/group?accountId=46897a8b-4ae8-45e2-b934-c4d00dba40a6" 
+```
+
+```java
+import org.killbill.billing.client.api.gen.InvoiceApi;
+protected InvoiceApi invoiceApi;
+
+UUID accountId = UUID.fromString("34a65013-2dd1-480e-b4b8-7999bb15ebce");
+LocalDate targetDate = LocalDate.parse("2023-06-15");
+Map<String, String> NULL_PLUGIN_PROPERTIES = null;
+
+Invoices invoices = invoiceApi.createFutureInvoiceGroup(accountId, targetDate, NULL_PLUGIN_PROPERTIES, requestOptions);
+```
+
+```ruby
+TODO
+```
+
+```python
+TODO
+```
+> Example Response:
+
+```shell
+# Subset of headers returned when specifying -v curl option
+< HTTP/1.1 201
+< Content-Type: application/json
+< Content-Length: 0
+< Location: http://127.0.0.1:8080/1.0/kb/invoices/60f5af97-9457-4b53-9eec-aec1de96a66f/group
+```
+
+```java
+TODO
```

Comment:
If you include the request for `java`, then we probably need the response as well.

Reply:
Fixed. I think there is actually an issue while retrieving the response when this endpoint is executed via the Java client. I had filed https://github.com/killbill/technical-support/issues/87 to investigate this further. For now, I've copied over the response from the `Retrieve Invoice Group` endpoint.